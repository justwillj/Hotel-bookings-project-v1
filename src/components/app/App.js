import Form from "../form/Form";
import NavBar from "../navBar/NavBar";
import { Routes, Route } from "react-router-dom";
import "./App.css";
import { BrowserRouter } from "react-router-dom/dist";
import Reservations from "../reservations/Reservations";
import CreateReservation from "../createReservation/CreateReservation";

function App() {
  return (
    <div>
      <NavBar />
      <BrowserRouter>
        <Routes>
          <Route exact path="/" element={<Form />}></Route>
          <Route exact path="/reservations" element={<Reservations />}></Route>
          <Route
            exact
            path="/reservations/create"
            element={<CreateReservation />}
          ></Route>
        </Routes>
      </BrowserRouter>
    </div>
  );
}

export default App;
