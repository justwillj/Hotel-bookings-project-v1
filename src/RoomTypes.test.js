import React from "react";
import {render,cleanup} from '@testing-library/react'
import App from "./components/app/App";
afterEach(cleanup)
 
it('should take a snapshot', () => {
   const { asFragment } = render(<App />)
   
   expect(asFragment(<App />)).toMatchSnapshot()
  })