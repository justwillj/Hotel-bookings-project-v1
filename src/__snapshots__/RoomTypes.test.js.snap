// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <nav>
      <div
        class="header"
      >
        <h2>
          Hotel Bookings
        </h2>
      </div>
      <ul />
    </nav>
    <div
      class="loading"
    >
      <div
        class="login-main"
      >
        <div
          class="main"
        >
          <form
            novalidate=""
          >
            <h1>
              Login
            </h1>
            <div
              class="input"
            >
              <label
                for="Email:"
              >
                Email:
              </label>
              <input
                id="Email:"
                name="Email:"
                type="email"
                value=""
              />
              <div
                class="error-message"
              >
                 
              </div>
            </div>
            <div
              class="input"
            >
              <label
                for="Password:"
              >
                Password:
              </label>
              <input
                id="Password:"
                name="Password:"
                type="password"
                value=""
              />
              <div
                class="error-message"
              >
                 
              </div>
            </div>
            <button
              class="login-btn"
              type="submit"
            >
              Login
            </button>
          </form>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
